server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/SieveMailAdapter.java

```diff
@@ -383,7 +385,7 @@ public boolean isInBodyText(List<String> phrasesCaseInsensitive) throws SieveMai
                 .contentTypes(Lists.newArrayList("text/plain"))
                 .includeHeaders(false)
                 .caseInsensitive(false)
-                .searchContents(Lists.transform(phrasesCaseInsensitive, s -> s)).build()
+                .searchContents(phrasesCaseInsensitive.stream().map(s -> s).collect(Guavate.toImmutableList())).build()
```

Comment:
Is the map necessary here?

Reply:
there's a generic mismatch problem but call ImmutableList.copy does the trick